cases.json

```diff
@@ -24,83 +24,120 @@
             }
         },
         {
-            "name" : "Connect-Compatible_Ciphers_1",
-            "client" : {
-                "min_version" : 770,
-                "max_version" : 772,
-                "cipher" : "TLS_AES_128_GCM_SHA256"
+            "name" : "Connect-TLS13-Ciphers",
+            "shared_params" : {
+              "ciphers" : [
+                "TLS_AES_128_GCM_SHA256;AES128-GCM-SHA256",
```

Comment:
Rather than include these a pairs (and risk having every new interop target have its own way of formatting names), can you instead use a common name and build a translator for each driver (or a translator *into* each driver) or is that too much to ask?

Reply:
I first tried to implement it the way you suggested, but decided against it at some point, most likely because the relation between the different naming schemes is not entirely consistent and therefore hard to turn into a simple translator function. But i can revisit this, since it would obviously be the more elegant solution.

Reply:
I was considering having a straight mapping, like here, except that each ciphersuite mapping would only appear once.  With this arrangement, you have the mapping every time the cipher is written.

Your translator function could use a switch statement or hash for those that don't fit the function.

Reply:
I combined the cipher name mapping and the blacklist in one file. It works well and should be easily extendable if more shims are added in the future.